github-sso-recipe.md

```diff
@@ -0,0 +1,24 @@
+# A recipe for GitHub SSO
+
+To authorize Semaphore to use GitHub for an organization with SSO enabled, there are two configurations you need to update in Github.
+
+1. Enable SSO for Semaphore SSH key `semaphore-key`.
+2. Grant orgnization access to Semaphore 2.0 deployment app.
+
+Below you'll find detailed steps on how to accomplish both steps.
+
+## Enable SSO for Semaphore SSH key
+1. Goto https://github.com/settings/profile
+2. From the left side menu click "[SSH and GPG keys]: https://github.com/settings/keys".
+3. Find the key with id "semaphore-key".
```

Comment:
Shouldn't the user need to create it first?

Reply:
I believe this key is automatically created once you connect Semaphore to your Github.

Reply:
Give me some more time to test adding the repository without Enabling SSO to `semaphore-key`.

Reply:
It looks like this step is unnecessary! @markoa . Updating the file.